lib/inspec/plugin/v2/installer.rb

```diff
@@ -461,9 +461,9 @@ def build_gem_request_universe(extra_request_sets = [], gem_to_force_update = ni
 
       # Combine the Sets, so the resolver has one composite place to look
       Gem::Resolver.compose_sets(
-        installed_plugins_gem_set,     # The gems that are in the plugin gem path directory tree
+        installed_plugins_gem_set, # The gems that are in the plugin gem path directory tree
         InstalledVendorSet.new,
-        *extra_request_sets,           # Anything else our caller wanted to include
+        *extra_request_sets # Anything else our caller wanted to include
```

Comment:
is the style application removing the trailing comma AND reformatting the alignment? I'm not sure I want the latter and would be willing to argue this one with lamont if necessary...